examples/05-all-for-one.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use React\Dns\Resolver\Factory;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$loop = React\EventLoop\Factory::create();
+
+$factory = new Factory();
+$resolver = $factory->create('8.8.8.8', $loop);
+
+$name = isset($argv[1]) ? $argv[1] : 'blog.wyrihaximus.net';
```

Comment:
In case anyone wonders why my own blog, because it is hosted on AWS CloudFront and this always returns more then one IP Address